.github/workflows/build.yml

```
@@ -169,9 +177,17 @@ jobs:
       run: |
         mkdir -p allure-results/history
         cp -r build-history-data/cht-core/allure/${{ matrix.cmd }}-${{ matrix.suite }}/allure-report/history allure-results | true
-    - name: Run tests ${{ matrix.cmd }}-${{ matrix.suite }}
+    - name: Install Chrome 90
+      if: ${{matrix.chrome-version=='90'}}
+      run: |
+        wget -O "google-chrome-stable_current_amd64.deb" https://www.slimjet.com/chrome/download-chrome.php?file=files/90.0.4430.72/google-chrome-stable_current_amd64.deb
+        sudo dpkg -i "google-chrome-stable_current_amd64.deb"
+    - name: Install Chromedriver 90
+      if: ${{matrix.chrome-version=='90'}}
+      run: npm install chromedriver@^90.0.1
```

Let's merge these two together, ie:

```suggestion
    - name: Install Chrome 90
      if: ${{matrix.chrome-version=='90'}}
      run: |
        wget -O "google-chrome-stable_current_amd64.deb" https://www.slimjet.com/chrome/download-chrome.php?file=files/90.0.4430.72/google-chrome-stable_current_amd64.deb
        sudo dpkg -i "google-chrome-stable_current_amd64.deb"
        npm install chromedriver@^90.0.1
```